tikv/kv.go

```diff
@@ -655,6 +672,40 @@ func (s *KVStore) updateSafeTS(ctx context.Context) {
 	wg.Wait()
 }
 
+var (
+	skipSafeTSUpdateCounter    = metrics.TiKVSafeTSUpdateCounter.WithLabelValues("skip", "cluster")
+	successSafeTSUpdateCounter = metrics.TiKVSafeTSUpdateCounter.WithLabelValues("success", "cluster")
+	clusterMinSafeTSGap        = metrics.TiKVMinSafeTSGapSeconds.WithLabelValues("cluster")
+)
+
+// setClusterMinSafeTSByPD check whether it is needed to get cluster-level's min resolved ts from PD.
+func (s *KVStore) setClusterMinSafeTSByPD(ctx context.Context) bool {
```

Comment:
This function's name sounds like a method that will set something, which is not, however.

Reply:
updated, what do u think :)